src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx

```diff
@@ -21,142 +21,153 @@ signupBanner:
-* **License key and app ID:** Near the bottom of the generated JavaScript is your browser license key and application ID. This is useful with the [REST API](/docs/apis/rest-api-v2/browser-examples-v2) and [API Explorer](/docs/apis/rest-api-v2/api-explorer-v2/getting-started-new-relics-api-explorer).
+<Step>
+### (Optional) Enable distributed tracing
 
-### Instrument manually on an APM-monitored app [#agent-instrumentation]
+Next, you can choose to enable distributed tracing. Read more about distributed tracing [here](/docs/browser/new-relic-browser/browser-pro-features/browser-data-distributed-tracing/).
 
-Our <InlinePopover type="apm" /> agents can instrument webpages with the required JavaScript for page load timing. If you are using an APM agent's API to manually add the JavaScript snippet to your webpages, insert the instrumentation snippet as close to the top as possible. This allows you to take advantage of detailed information about browser's [AJAX calls](/docs/browser/new-relic-browser/dashboard-details/ajax-dashboard) and [JavaScript errors](/docs/browser/new-relic-browser/dashboard-details/javascript-errors-dashboard).
+<img
+  title="enable distributed tracing"
+  alt="enable distributed tracing"
+  src={browserInstallStep3}
+  style={{ maxWidth: '70%' }}
+/>
+</Step>
 
-For more information, see the instructions for your APM agent:
+<Step>
+### Name your browser app
 
-* [Go](/docs/agents/go-agent/features/install-new-relic-browser-go-apps)
-* [Java](/docs/agents/java-agent/instrumentation/page-load-timing-java)
-* [.NET](/docs/agents/net-agent/features/page-load-timing-net)
-* [Node.js](/docs/agents/nodejs-agent/supported-features/page-load-timing-nodejs)
-* [PHP](/docs/agents/php-agent/features/page-load-timing-php)
-* [Python](/docs/agents/python-agent/supported-features/page-load-timing-python)
-* [Ruby](/docs/agents/ruby-agent/features/page-load-timing-ruby)
+When selecting a name for your browser app, you've got two options: 
 
-### Create a browser app with NerdGraph API [#create-browser]
+* Select an existing app already monitored by an APM agent. 
+* Create a new name for a standalone browser app. 
 
-Use the NerdGraph API to create a browser app manually. See this [Nerdgraph tutorial](/docs/apis/nerdgraph/examples/browser-monitoring-config-nerdgraph#create-browser). 
+Choose one of these options and click **Enable**. 
 
-<InstallFeedback />
+<img
+  title="name or connect an existing browser app"
+  alt="name or connect an existing browser app"
+  src={browserInstallStep4}
+  style={{ maxWidth: '70%' }}
+/>
+</Step>
 
-## Browser agent types: Lite, Pro, Pro+SPA [#agent-types]
+<Step>
+### (For copy/paste method only) Inject the JavaScript snippet
 
-We have three types of browser agents: Lite, Pro, and Pro+SPA. The agent type has no impact on your billing.
+If you selected the copy/paste option, it's time to add the JavaScript snippet to each of your browser app webpages' HTML. 
```

Comment:
The phrasing of this sentence feels slightly awkward.

maybe something more like "Add the snippet to each individual page of the web app you want to monitor." or something